temp.py

```diff
@@ -0,0 +1,2 @@
+import pdb
```

Comment:
Avoid using `pdb` for debugging in committed code. It's better to remove it or use proper logging for debugging purposes.
```
import logging

# Set up logging
logging.basicConfig(level=logging.DEBUG)
logger = logging.getLogger(__name__)

# Example usage
logger.debug('Debug message')
```